src/service-worker/ServiceWorker.ts

```diff
@@ -345,6 +345,9 @@ export class ServiceWorker {
     if (!notification)
       return null;
 
+    if (!ServiceWorker.browserSupportsConfirmedDelivery())
```

Comment:
Nit: use { }

Reply:
Left for now, since the `if` above and below does not have `{}`. Preferring consistency in the function over following the standard.

If this PR had more changes I would consider updating all the `if`s.